klippa/src/layout.rs

```diff
@@ -365,14 +368,16 @@ fn classdef_remap_and_serialize(
 }
 
 trait Serialize<'a> {
+    type ArgsForSerialize: 'a;
```

Comment:
you can probably just call the type `Args`, it will be clear from context what args we're talking about?